jwst/pipeline/calwebb_spec2.py

```diff
@@ -292,12 +290,14 @@ def process_exposure_product(
 
             # Call the resample_spec step for 2D slit data
             resampled = calibrated.copy()
+            resampled = self.pixel_replace(resampled)
             resampled = self.resample_spec(resampled)
 
         elif is_nrs_slit_linelamp(calibrated):
 
             # Call resample_spec for NRS 2D line lamp slit data
```

Comment:
This comment needs updating or moving too.

Reply:
done